spec/models/notification_spec.rb

```diff
@@ -19,9 +19,14 @@
         expect(user.unseen_notifications.count).to eq(1)
       end
 
-      it 'broadcasts the message through ActionCable' do
-        expect_any_instance_of(ActionCable::Server::Base).to receive(:broadcast)
-        subject # force the creation of the db object
+      context 'if asynchronous notifications are enabled' do
+        # Do not let this test fail locally when someone disables the notifications
+        if ::Settings.server.asynchronous_notifications
```

Comment:
I would suggest to use ```stub_settings(:server => {:asynchronous_notifications => true})```
Also, it make sense to do 2 expectation with this settings set  to both ```true``` and ```false```